fs/fs_zenfs.cc

```diff
@@ -652,6 +652,11 @@ void ZenFS::GetZenFSChildrenNoLock(const std::string& dir,
   std::string dir_with_terminating_seperator =
       path_as_string_with_separator_at_end(std::filesystem::path(dir));
 
+  auto relative_child_path =
+      [dir_with_terminating_seperator](std::string const& full_path) {
```

Comment:
```suggestion
      [&dir_with_terminating_seperator](std::string const& full_path) {
```
No need to copy the path, it is probably fine to take a reference.

Reply:
Fixed :)